### PR TITLE
Add script for payment import from Fio CZ

### PR DESCRIPTION
**Pull request by enumag**

### CODE REVIEW

File: config.dist.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+define('TEMP_DIR', __DIR__ . '/temp');
+
+const UCRM_API_VERSION = '1.0';
+const UCRM_API_URL = '';
+const UCRM_API_KEY = '';
+
```
Comment (UBNT-Petr) [inline config.dist.php]: what about to separate config for UCRM API and FIO API?

File: config.dist.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+define('TEMP_DIR', __DIR__ . '/temp');
+
+const UCRM_API_VERSION = '1.0';
+const UCRM_API_URL = '';
```
Comment (UBNT-Petr) [inline config.dist.php]: add axample as a comment: https://ucrm-demo.ubnt.com

File: config.dist.php [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+const UCRM_API_KEY = '';
```
Comment (UBNT-Petr) [inline config.dist.php]: add comment: `Get your key in System > Security > App keys`

File: config.dist.php (hunk 1 above)
Comment (UBNT-Petr) [reply config.dist.php]: + some link to FIO API docs?

**Review by UBNT-Petr — CHANGES_REQUESTED** [review]

File: sdk.php [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    curl_setopt($c, CURLOPT_URL, $url);
+    curl_setopt($c, CURLOPT_HTTPHEADER, $headers);
+    curl_setopt($c, CURLOPT_CUSTOMREQUEST, $method);
+
+    if ($data) {
+        curl_setopt($c, CURLOPT_POST, true);
+        curl_setopt($c, CURLOPT_POSTFIELDS, $data);
+    }
+
+    curl_setopt($c, CURLOPT_RETURNTRANSFER,true);
```
Comment (janprochazkacz) [inline sdk.php]: indent

**Review by janprochazkacz — CHANGES_REQUESTED** [review]

**Pull request merged**